packages/api/amplify_api/test/graphql_test.dart

```diff
@@ -100,7 +100,7 @@ const _expectedAuthErrorResponseBody = {
   ]
 };
 
-final mockHttpClient = MockAWSHttpClient((request) async {
+final mockHttpClient = MockAWSHttpClient((request, RunUnaryHandler) async {
```

Comment:
The second param is an `isCancelled` getter to check if `request` has been cancelled, e.g.

```dart
MockAWSHttpClient((request, isCancelled) async {
  if (isCancelled()) return;
  // ...
});
```

Best practice is to use `_` for unused variables to indicate they aren't being used.

```suggestion
final mockHttpClient = MockAWSHttpClient((request, _) async {
```

Reply:
thanks, made this change throughout